### DISCUSSION TITLE
How to change the width of a QEditor slot?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#9561] [General - Components / Directives / etc] [answered]
I am using (probably misusing :neutral_face:) the toolbar of a QEditor to add a `<q-input>`

```
<q-editor
          model-value="undefined"
          v-model="note.text"
          @click="noteClicked(note)"
          @blur="noteBlurred(note)"
          :toolbar="[['title'], ['tags']]"
      >
        <template v-slot:title>
          <q-input model-value="undefined" v-model="note.title" autofocus dense />
        </template>
        <template v-slot:tags>
          <q-chip v-for="tag in note.tags" removable @remove="removeTag(note, tag)" color="primary" text-color="white" :label="tag">
          </q-chip>
        </template>
      </q-editor>
```

The codepen.io for the above is here: https://codepen.io/WoJWoJ/pen/yLMKWvJ

I would like to make the `title` slot wider - how can I do that?

**Comment by ontwikkelfabriek** [comment]
@wsw70 

Please create a codepen.io so we can provide better support.

**Reply by wsw70** [reply]
@ontwikkelfabriek done! Thanks

**Reply by ontwikkelfabriek** [reply]
@wsw70 

> I am using (probably misusing 😐) 

Yes I don't think you want to put your title there. The text in the toolbar now has a onfocus blue underline bar style when you click on it. You could css hack it away I am sure.   Why not put the title above the editor?

Did you look at this editor:
https://github.com/ueberdosis/tiptap

**Reply by wsw70** [reply]
> Why not put the title above the editor?

I wanted to make it as compact as possible. The underline is not a problem - what I am really after is to make the area wider.

**Reply by wsw70** [reply]
> https://github.com/ueberdosis/tiptap

Oh, it looks great, and it would be easy to configure for my needs. Thanks!

**Comment by wsw70** [comment, marked as answer]
The solution was to add a class with `width: 30vw`
